.github/dependabot.yml

```diff
@@ -0,0 +1,15 @@
+# Please see the documentation for all configuration options:
+# https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "pip"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+    rebase-strategy: "disabled"
```

Comment:
having this enabled/auto can get annoying when there are many PRs open as it will cause a cascading effect:
- 4 PRs opened by dependabot
- PR-1 is merged
- 3 PRs get rebased automatically
- PR-2 is merged
- 2 PRs get rebased automatically

(and so on) - disabling means the project maintainer can ask dependabot to rebase the PRs on-demand when it's getting ready to approve/merge